src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/VerifyClientDeviceIdentityTest.java

```diff
@@ -120,85 +138,63 @@ private void startNucleusWithConfig(String configFileName, State expectedService
         kernel.getContext().removeGlobalStateChangeListener(listener);
     }
 
-    @AfterEach
-    void afterEach() {
-        LogConfig.getRootLogConfig().reset();
-        kernel.shutdown();
-    }
-
     @Test
-    void GIVEN_broker_WHEN_verify_client_identity_THEN_verify() throws Exception {
+    void GIVEN_requestWithValidCertificate_WHEN_verifyClientIdentity_THEN_returnValid() throws Exception {
         kernel.getContext().put(IotAuthClient.class, iotAuthClient);
-        when(iotAuthClient.getActiveCertificateId(anyString())).thenReturn(
-                Optional.of("SOME-CERT")
-        );
+        clientCertificate.setStatus(Certificate.Status.ACTIVE);
+        when(iotAuthClient.getIotCertificate(validClientCertificatePem)).thenReturn(Optional.of(clientCertificate));
```

Comment:
NOTE: this test is still more fragile than I'd like since we're mocking the iot auth client.

We have a task to write a iot auth client fake which we can use with tests and we'll migrate to that afterward